ISSUE_TEMPLATE/whitelist_template.md

```diff
@@ -0,0 +1,36 @@
+# Whitelist Request
+- Name of the game to be whitelisted:
+- Steam AppID of the game: 
+
+## System Information
+- Driver version<!-- e.g. Mesa 18.2.0-devel -->:
+- Kernel version<!-- e.g. 4.17 -->:
+- LLVM version [optional]<!-- e.g. 7.0.0 -->:
+- Graphics processor<!-- e.g. RX 580 -->:
+- Link to full system information report as [Gist](https://gist.github.com/):
+- Proton version:
+
+## I confirm:
+- [ ] that pressing the `Play` button in the Steam client is sufficient.
+- [ ] that no workaround has to be made (e.g. renaming game files, preloading
```

Comment:
If I'm understanding the requirements correctly, if only Proton [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options), then the game can still be considered for whitelisting.